src/dodal/devices/i04/murko_results.py

```diff
@@ -42,12 +41,19 @@ class Coord(Enum):
 
 
 class MurkoResultsDevice(StandardReadable, Triggerable, Stageable):
-    """Device that takes crystal centre coords from Murko and uses them to set the
+    """Device that takes crystal centre values from Murko and uses them to set the
     x, y, z coordinate of the sample to be in line with the beam centre.
-    (x, z) coords can be read at 90°, and (x, y) at 180° (or the closest omega angle to
-    90° and 180°). The average of the x values at these angles is taken, and sin(omega)z
-    and cosine(omega)y are taken to account for the rotation. This value is used to
-    calculate a number of mm the sample needs to move to be in line with the beam centre.
+    The most_likely_click[1] value from Murko corresponds with the x coordinate of the
+    sample. The most_likely_click[0] value from Murko corresponds with a component of
+    the y and z coordinates of the sample, depending on the omega angle, as the sample
+    is rotated around the x axis.
+
+    Given a most_liekly_click value at a certain omega angle θ:
```

Comment:
Nit: 
```suggestion
    Given a most_likely_click value at a certain omega angle θ:
```
Do you have spell check on vscode? I find it very useful